app/src/main/java/org/kiwix/kiwixmobile/main/KiwixMainActivity.kt

```diff
@@ -87,6 +87,9 @@ class KiwixMainActivity : CoreMainActivity() {
         handleDrawerOnNavigation()
       }
     }
+    if (sharedPreferenceUtil.showIntro()) {
+      navigate(R.id.introFragment)
```

Comment:
if I press back on Intro I go to the main view, only "Get Started" should do it

Reply:
ah, right!